_docs/_latest/configure/config-validator/policy-library-sync-from-git-repo.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Policy Library Sync from Git Repository
+order: 702
+---
+
+# {{ page.title }}
+
+This page describes how to sync policies from Git repository.
+
+---
+
+## **Policy Library Sync from Git Repository**
+
+As part of the Terraform configuration, you will need to include a few 
+additional variables to enable the git-sync feature.
+ 
+In your `main.tf` file, set the `policy_library_sync_enabled` variable in the 
```

Comment:
I think users will need to know how to setup the Policy Library repo. Unfortunately I think we will need to link to Github or explore using a gist. https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-set-up-constraints-with-policy-library

Reply:
Done, linked to the Policy Library repo.